### PR TITLE
September Components Sprint

### PR DESCRIPTION
**Pull request by kaladay**

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -8,3 +8,8 @@
 }
 
 @import '../projects/wvr-elements/src/lib/shared/styles/wvr-elements';
+
+html,
+body {
+  height: 100%;
```
Comment (jeremythuff) [inline src/styles.scss:14]: I do not think we should be imposing site wide styles from a component

**Review by jeremythuff — CHANGES_REQUESTED** [review]

File: src/styles.scss (hunk 1 above)
Comment (wwelling) [reply src/styles.scss:14]: I do. Lets do it. But set it when a component is registered.

Comment (wwelling) [reply src/styles.scss:14]: What would break?

Comment (kaladay) [reply src/styles.scss:14]: I wonder, should this be moved to tamu-library-components CSS/SCSS?

**Review by mumptastic — APPROVED** [review]

**Review by rmathew1011 — APPROVED** [review]

**Review by jeremythuff — APPROVED** [review]

**Pull request merged**